nnet_testing_script.m

```diff
@@ -0,0 +1,114 @@
+% Load in the classification model
+
+modelDir = "../models/";
+modelName = "nnet";
+load(modelDir + modelName);
+% Add path to use extractFeatures
+
+addpath ../insect-lidar-supervised-classification/
+% Find all relevant files
+
+dataDir = "../2020-08-13/";
+files = dir(dataDir + "/*/adjusted_data_decembercal.mat"); %finding all relevant files
+% Create a struct to easily identify flagged insect images
+
+%struct array for storing filename and labels for images with insects
+
+nnetInsectImages = struct('fileName', 'None', 'labels', zeros(178,1,"logical"), 'isInsect', 'None', 'insectRows', []);
+i = 1;
+a = 1;
+nnetInsectImageFilesArray = string.empty;
+insectExists = "false";
+% Loop through all images from all the files and add a row to the struct
+
+for k = 1:length(files)
+    %grab the next file
+    currFileName = files(k).name;
+    currFolder = files(k).folder;
+    currFileFullName = fullfile(currFolder, currFileName);
+
+    %grab the data for the current file
+    matData = load(currFileFullName);
+    
```

Comment:
We'll need to transform the manual labels into an array that we can actually use. We can do that here for each adjusted_data_decembercal file. Check out `combineScans.m` to see how we've done it before.

I think this will work:
```matlab
manualLabels = extractLabels(manual.insects, matData.adjusted_data_decembercal);
manualImageLabels = cellfun(@(labels) any(labels), manualLabels);
```

We'll need to concatenate all the image labels for each folder together.

We should also keep track of the "maybe insect" labels. We can do the same thing as above to get the maybe insect labels:
```matlab
manualLabels = extract(manual.maybe_insect, matData.adjusted_data_decembercal);
manualImagesMaybeInsects = cellfun(@(labels) any(labels), manualLabels);
```

Reply:
@tvannoy There seems to be an error in that first line of code and I can't figure out how to view what it is in matlab, do you have any suggestions?

Reply:
My first guess is that [extractLabels](https://github.com/BMW-lab-MSU/insect-lidar-supervised-classification/blob/main/extractLabels.m) might not be on your path.

The problem might also be that `manual` isn't defined; I noticed that you never explicitly loaded in the manual labels mat file in the script, which is a problem regardless.